repose-aggregator/components/services/rate-limiting-service/src/test/groovy/org/openrepose/core/services/ratelimit/RateLimitingServiceImplTest.groovy

```diff
@@ -89,8 +94,12 @@ public class RateLimitingServiceImplTest extends RateLimitServiceTestContext {
 
         queryParamLimitGroup.getLimit().add(newLimitConfig("query-param-test", "*", ".*", methods, ["index"].asList()))
 
+        defaultMethodsLimitGroup.getLimit().add newLimitConfig("methods-test-empty", "/methods/empty/*", "/methods/empty/.*", Collections.emptyList(), queryNames)
+        defaultMethodsLimitGroup.getLimit().add newLimitConfig("methods-test-all", "/methods/all/*", "/methods/all/.*", Collections.singletonList(HttpMethod.ALL), queryNames)
```

Comment:
Yep. I don't know why I did that or if some weird formatting thing did it. Updated.